fwb-plugin/src/main/java/sub/fwb/api/ImporterStepRunTests.java

```diff
@@ -32,11 +44,56 @@ public void execute(Map<String, String> params) throws Exception {
 			out.println();
 			out.println("WARNING in " + fail.getTestHeader() + ": " + fail.getMessage());
 		}
+		clearCache();
 	}
 
 	@Override
 	public String getStepDescription() {
 		return "Testabfragen in Solr";
 	}
+	
+	/**
+	 * Clearing the FWB API cache after finish indexing
+	 * @throws IOException
+	 */
+	private void clearCache() throws IOException {
+		try {
+			String webPage = "https://fwbdev.sub.uni-goettingen.de/api/cache/clear";
```

Comment:
Please don't put this URL into the code. Make it configurable. It should also not be visible as it may result in bots clearing the cache and cause a DDOS attack